script.js

```diff
@@ -476,12 +567,15 @@ webix.ready(function() {
     rows: [header, content, footer]
   });
 
-  $$("usersChart").sync($$("usersList"), function() {
+  $$("usersList").sync(usersCollection);
+  $$("usersChart").sync(usersCollection, function() {
     this.group({
       by: "country",
       map: {
         age: ["id", "count"]
       }
     });
   });
+  $$("addCategoryForm").bind($$("categoryDataTable"));
```

Comment:
bind does not suit here, because `form.save` add items to the bound widget, but you need to add it to a collection.

- using bind, added data is not synced with combo and column filter in the 1st tab
- moreover, when you remove the category by clicking the trash icon, you remove it from collection. and it throws an error as there's no item with such id in it

Reply:
also, did not find Richselect with categories in the form (first tab)

Reply:
Ready to review